.github/workflows/test.yml

```diff
@@ -575,6 +575,23 @@ jobs:
         name: kind-logs-${{ env.JOB_NAME }}-${{ github.run_id }}
         path: /tmp/kind/logs
 
+    - name: Export ovn dbs
```

Comment:
Note: this is needed here because failure may happen in the bash script. If it happened in e2e test, then the better approach would have been to use [wrappedTestFramework](https://github.com/ovn-org/ovn-kubernetes/blob/86f9e1fd9439527eb12b3661c7e4feda311ab217/test/e2e/util.go#L900) via ginko.